include/mpc/LMPC.hpp

```diff
@@ -134,18 +134,9 @@ namespace mpc
             const cvec<Tnx> XMax, const cvec<Tnu> UMax, const cvec<Tny> YMax,
             const std::array<int, 2> slice)
         {
-            size_t start = slice[0];
```

Comment:
This was a lazy implementation, could be useful to put these checks in a shared function call

Reply:
@Veng97 if we solve this then we can merge